src/components/ViewedByCustomer/ViewedByCustomer.ts

```diff
@@ -0,0 +1,65 @@
+import { Component, ComponentOptions, IComponentBindings, $$, Initialization } from 'coveo-search-ui';
+import { user } from '../../utils/icons';
+
+/**
+ * The options for the ViewedByCustomerComponent
+ */
+export interface IViewedByCustomerOptions {
+    showIcon?: boolean;

```

Comment:
Juicy comments on each options

Reply:
Do it in the type so intellisense can catch it

Reply:
One or the other, not both, tho: DRY.